common/src/main/java/com/wynntils/gui/render/Texture.java

```diff
@@ -50,10 +50,11 @@ public enum Texture {
     SETTINGS_ICON("settings_icon.png", 17, 34),
 
     // Map related
-    GILDED_MAP_TEXTURES("map/gilded_map_textures.png", 262, 524),
-    PAPER_MAP_TEXTURES("map/paper_map_textures.png", 256, 512),
-    WYNN_MAP_TEXTURES("map/wynn_map_textures.png", 256, 256),
-    FULLSCREEN_MAP_BORDER("map/full_map_border.png", 510, 254),
+    GILDED_MAP_TEXTURES("map/map_border/gilded_map_textures.png", 262, 524),
+    PAPER_MAP_TEXTURES("map/map_border/paper_map_textures.png", 256, 512),
+    WYNN_MAP_TEXTURES("map/map_border/wynn_map_textures.png", 126, 256),
+    CIRCLE("map/map_border/circle.png", 256, 256),
+    FULLSCREEN_MAP_BORDER("map/map_border/full_map_border.png", 510, 254),
```

Comment:
Can you order this whole map segment alphabetically? Everything else is ordered iirc

Reply:
@kristofbolyai That was never done before integration..?

Reply:
Oh I forgot. We can just do it next time I touch the map